deepchem/feat/molecule_featurizers/roberta_tokenizer.py

```diff
@@ -0,0 +1,39 @@
+from deepchem.feat import MolecularFeaturizer
+from transformers import RobertaTokenizerFast
+from deepchem.utils.typing import RDKitMol
+
+from typing import Dict, List
+
+
+class RobertaFeaturizer(RobertaTokenizerFast, MolecularFeaturizer):
```

Comment:
Should we just call this RobertaTokenizer to match the SmilesTokenizer naming convention?

Reply:
I think we can try to keep it RobertaFeaturizer since we plan to phase SmilesTokenizer eventually once this gains adoption right?